### ISSUE TITLE
Initialization exception on windows 7

### ISSUE THREAD

**Issue by iddl** [DataDog/datadogpy#47]
Initializing datadogpy on windows7 results in the exception below.

Note: running python 3.4

```
Traceback (most recent call last):
  File "<pyshell#1>", line 1, in <module>
    datadog.initialize()
  File "C:\Python34\lib\site-packages\datadog\__init__.py", line 59, in initialize
    api._host_name = host_name if host_name is not None else get_hostname()
  File "C:\Python34\lib\site-packages\datadog\util\hostname.py", line 52, in get_hostname
    config = get_config()
  File "C:\Python34\lib\site-packages\datadog\util\config.py", line 122, in get_config
    config_path = get_config_path(cfg_path, os_name=get_os())
  File "C:\Python34\lib\site-packages\datadog\util\config.py", line 101, in get_config_path
    return _windows_config_path()
  File "C:\Python34\lib\site-packages\datadog\util\config.py", line 68, in _windows_config_path
    common_data = _windows_commondata_path()
  File "C:\Python34\lib\site-packages\datadog\util\config.py", line 62, in _windows_commondata_path
    path_buf = wintypes.create_unicode_buffer(wintypes.MAX_PATH)
AttributeError: 'module' object has no attribute 'create_unicode_buffer'
```

**Comment by samanthadrago** [comment]
Feedback from customer - it works if using python 2.7. We could add a note to the README?

**Comment by yannmh** [comment]
Thanks @samanthadrago. Do you know which version they were using ?

**Comment by samanthadrago** [comment]
@yannmh 3.4

**Comment by yannmh** [comment]
Thanks @samanthadrago. 
Datadogpy should work with Python 3.4 as listed in [pypi classifiers](https://github.com/DataDog/datadogpy/blob/master/setup.py#L47), I'll investigate on a Windows lab to figure out what the issue is.

**Comment by apoorvab** [comment]
@yannmh, I think I found a solution. I tested it out and it worked for me.  

According to a comment by eryksun in "http://stackoverflow.com/questions/626796/how-do-i-find-the-windows-common-application-data-folder-using-python" (provided in the code) ctypes.create_unicode_buffer should be used rather than wintypes.create_unicode_buffer. 

This is because "[create_unicode_buffer] was only accidentally in wintypes in PY2 due to its use of from ctypes import *"

I was on Windows 7 using Python 3.4 in a virtual environment with PyCharms as my IDE.
